lib/rack/honeycomb/middleware.rb

```diff
@@ -110,6 +110,10 @@ def debug(msg)
       end
 
       def add_request_fields(event, env)
+        event.add_field('name', "#{env['REQUEST_METHOD']} #{env['PATH_INFO']}")
+        # N.B. 'name' may be overwritten later by add_sinatra_fields or
+        # add_rails_fields
+
```

Comment:
if we don't continue to add this here then if you are using this gem standalone (without the beeline) it will never be set